Refresh.GameServer/Types/OAuth/OAuth2AccessTokenResponse.cs

```diff
@@ -0,0 +1,15 @@
+using JetBrains.Annotations;
+
+namespace Refresh.GameServer.Types.OAuth;
+
+#nullable disable
```

Comment:
Same here, except somehow we're also using the correct `?` syntax and `[CanBeNull]` at the same time..?